models/intermediate/int__observation_allergies.sql

```diff
@@ -5,7 +5,6 @@ SELECT
     , srctostdvm.target_concept_id AS observation_concept_id
     , a.allergy_start_date AS observation_date
     , a.allergy_start_date AS observation_datetime
-    , 32827 AS observation_type_concept_id
```

Comment:
simplifying observation models by moving this col to the final mart model (also, the type concept ID was misspecified with a nonstandard concept in int__observation_conditions.